modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -1133,8 +1137,12 @@ void GDScriptAnalyzer::resolve_function_signature(GDScriptParser::FunctionNode *
 		is_shadowing(p_function->parameters[i]->identifier, "function parameter");
 #endif // DEBUG_ENABLED
 #ifdef TOOLS_ENABLED
-		if (p_function->parameters[i]->default_value && p_function->parameters[i]->default_value->is_constant) {
-			p_function->default_arg_values.push_back(p_function->parameters[i]->default_value->reduced_value);
+		if (p_function->parameters[i]->default_value) {
+			default_value_count++;
+
+			if (p_function->parameters[i]->default_value->is_constant) {
+				p_function->default_arg_values.push_back(p_function->parameters[i]->default_value->reduced_value);
+			}
```

Comment:
This seems to be wrong, it should put all the default values. It doesn't work properly like this, since the consumer won't be able to tell which values belong to each parameter. But for what I know, this is only used in the generated documentation, that's probably why nobody noticed.

I know this isn't your code, it's just what was there before, but I wanted to point it out. I can't think of a good fix for now, so let's leave it like this. I'll make a note in my list to test and open an issue about it.